pkg/plugins/backendplugin/grpcplugin/client.go

```diff
@@ -47,16 +47,12 @@ func newClientConfig(executablePath string, env []string, logger log.Logger,
 	}
 }
 
-// LegacyStartFunc callback function called when a plugin with old plugin protocol is started.
-type LegacyStartFunc func(pluginID string, client *LegacyClient, logger log.Logger) error
-
 // StartFunc callback function called when a plugin with current plugin protocol version is started.
 type StartFunc func(pluginID string, client *Client, logger log.Logger) error
 
 // PluginStartFuncs functions called for plugin when started.
 type PluginStartFuncs struct {
-	OnLegacyStart LegacyStartFunc
-	OnStart       StartFunc
+	OnStart StartFunc
```

Comment:
After consideration I think we should remove everything related/referencing `DefaultProtocolVersion` in this file and change `handshake.ProtocolVersion = grpcplugin.ProtocolVersion`. With this we should get rid of the last reference to github.com/grafana/grafana-plugin-model and we can tidy up the go.mod file to get rid of the github.com/grafana/grafana-plugin-model dependency altogether.

Reply:
I removed the DefaultProtocolVersion as well from renderer and backend plugin